vm/src/exceptions.rs

```diff
@@ -488,6 +488,7 @@ pub struct ExceptionZoo {
     pub indentation_error: PyTypeRef,
     pub tab_error: PyTypeRef,
     pub system_error: PyTypeRef,
+    pub struct_error: PyTypeRef,
```

Comment:
is this same error as the `struct.struct_error`? Then `pystruct.rs` has `new_struct_error`.

If not, still we were keeping this place only for built-in exceptions until now. Maybe adding built-in modules exception here could be a nice idea. But I want to know if it could affect to compatibility somehow.